### DISCUSSION TITLE
How to change profileMode's default picture changes?

### DISCUSSION THREAD

**Discussion by eangutierrez** [adityatelange/hugo-PaperMod#1279] [Q&A]
Hi everyone, 

Is there a way to stop profileMode from automatically changing profile pictures? I have a couple of pictures that I want to post as they are ( without any rounding or dimension changes), but I am not able to find the answer.  Thank you!

**Comment by chrisrummel** [comment]
Have a look at themes/hugo-theme-papermod/assets/css/common/profile-mode.css and copy it to assets/css/common/profile-mode.css to make changes.

**Reply by eangutierrez** [reply]
Hi Chris. Thank you for pointing me in the right direction.  I was able to find the profile-mode.css file.  The profile img has 3 statements: 
{
    display: inline-table;
    border-radius: 50%;
    pointer-events: none;
}

I got rid of the rounding by setting border-radius to 50%, but the picture is still small.  Would that also be fixed on the profile-mode.css file?
